src/Api/Api.csproj

```diff
@@ -32,5 +33,20 @@
     <PackageReference Include="Azure.Messaging.EventGrid" Version="4.10.0" />
     <PackageReference Include="Swashbuckle.AspNetCore" Version="6.3.1" />
   </ItemGroup>
-  
+
+  <Target Name="SetSourceRevisionId" BeforeTargets="CoreGenerateAssemblyInfo">
+    <Exec Command="git describe --long --always --dirty --exclude=* --abbrev=8" ConsoleToMSBuild="True" IgnoreExitCode="False">
+      <Output PropertyName="SourceRevisionId" TaskParameter="ConsoleOutput" />
+    </Exec>
+  </Target>
+
+  <Target Name="WriteRevision" AfterTargets="SetSourceRevisionId">
```

Comment:
Should we add this to all our projects, as it will help with tracking the git hash for each binary.

Reply:
Yes, especially if we end up making an Assembly Helper class like you mentioned [here](https://github.com/bitwarden/server/pull/2130#discussion_r927397306), we could potentially move the build configuration to `Directory.Build.props` and make `gitHash` available everywhere. Thoughts?

Reply:
Sounds like a good strategy.

Reply:
Updated.